src/controller/testController.ts

```diff
@@ -311,12 +303,22 @@ function handleInvocations(testItems: TestItem[]): TestItem[] {
     // remove invocations if they are already included in selected higher-level tests
     testItems = testItems.filter((item: TestItem) => !(isInvocation(item) && isAncestorIncluded(item, testItems)));
```

Comment:
Maybe put this line into the `mergeInvocations()` as well? Since it's dosing a kind of merging operations